index.bs

```diff
@@ -642,45 +681,6 @@ The {{VRDisplay}} associated with this event.
 <dfn attribute for="VRDisplayEvent">reason</dfn>
 {{VRDisplayEventReason}} describing why this event has has been fired.
 
-
-## Window Interface extension ## {#interface-window}
-
-<pre class="idl">
-partial interface Window {
-  attribute EventHandler onvrdisplayconnect;
-  attribute EventHandler onvrdisplaydisconnect;
-  attribute EventHandler onvrdisplayactivate;
```

Comment:
As at @domenic suggests, the way to do this is to add "attribute EventHandlers".  The HTML 5 spec does this for events like onload, onplaying and many others.